shoov/modules/custom/shoov_restful/plugins/restful/node/js_lm_incidents/1.0/ShoovJsLmIncidentsResource.class.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+/**
+ * @file
+ * Contains ShoovJsLmIncidentsResource.
+ */
+
+class ShoovJsLmIncidentsResource extends \ShoovEntityBaseNode {
+
+  /**
+   * Overrides \RestfulEntityBaseNode::publicFieldsInfo().
+   */
+  public function publicFieldsInfo() {
+    $public_fields = parent::publicFieldsInfo();
+
+    $public_fields['build'] = array(
+      'property' => 'field_js_lm_build',
+    );
+
+    $public_fields['user_id'] = array(
+      'property' => 'field_user_id',
+    );
+
+    $public_fields['errors'] = array(
+      'property' => 'field_js_lm_errors',
+    );
+
+    $public_fields['image'] = array(
+      'property' => 'field_js_lm_image',
+      'process_callbacks' => array(
+        array($this, 'imageProcess'),
+      ),
+    );
+
+    return $public_fields;
+  }
+
+  /**
+   * Overrides \ShoovEntityBaseNode::checkEntityAccess().
+   *
+   * Always grant access to create.
+   *
+   * @todo: Reconsider.
+   */
+  protected function checkEntityAccess($op, $entity_type, $entity) {
+    return TRUE;
```

Comment:
Seems like some token should be sent and checked since anonymous user is creating incident.